SymPEP-0001.md

```diff
@@ -0,0 +1,222 @@
+# SymPEP 1 --- SymPEP Purpose and Process
+
+**Author** Aaron Meurer
+**Status** Draft
+**Type** Process
+**Created** 2021-01-27
+**Resolution**
```

Comment:
I think that the headings don't look good in display though. You need to add some spacings